L1Trigger/Phase2L1ParticleFlow/plugins/L1TCorrectedPFJetProducer.cc

```diff
@@ -44,11 +44,12 @@ void L1TCorrectedPFJetProducer::produce(edm::StreamID, edm::Event& iEvent, const
     out->emplace_back(srcjet.p4());
     auto& jet = out->back();
     // copy daughters
-    //if (copyDaughters_) {
-    //  for (const auto& dau : srcjet.daughterPtrVector()) {
-    //    jet.addConstituent(edm::Ptr<l1t::L1Candidate>(dau));
-    //  }
-    //}
+    bool copyDaughters_ = true;
```

Comment:
I think you can remove the `if` entirely (it made sense there when the feature was configurable, but it's not the case now)

Reply:
Done.